mmpy_bot/bot.py

```diff
@@ -28,12 +30,18 @@ def __init__(
         self.settings = settings or Settings()
         logging.basicConfig(
             **{
-                "format": "[%(asctime)s] %(message)s",
+                "format": self.settings.LOG_FORMAT,
                 "datefmt": "%m/%d/%Y %H:%M:%S",
                 "level": logging.DEBUG if self.settings.DEBUG else logging.INFO,
-                "stream": sys.stdout,
+                "filename": self.settings.LOG_FILE,
+                "filemode": "w",
             }
         )
```

Comment:
Will the format be used globally? Or only for the logger in this file?

Reply:
The current implementation is that the LOG_FORMAT from settings is used for both the console login to stdout and the log file. However, if you think we should provide independent FORMAT settings for each I am fine with that. My goal was to provide some flexibility while still having a sane default.